cypress/e2e/ModelizerPage/TextView/file/update.file.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Test modelizer text view: update file's content
```

Comment:
You never check if the file content is set, add check on file content

Reply:
Done + add test to check if updated content is still set after switching tab